locales/fr.json

```diff
@@ -74,6 +74,7 @@
   },
   "collections": {
     "general": {
+      "all_of_collection": "Tous les {{ collection }}",
```

Comment:
For French it's definitely a show-stopper, so you can safely not reference the passed collection. 50% of times the sentence will read very bad, not just a little bad. So, in French translation use:

       "all_of_collection": "Tout",

That is like `All`. It is not weird, it just works all the time. Just using this:

       "all_of_collection": "{{ collection }}",

... will be confusing imho. How do you un-filter? How do you see all products? What is a sub-category and what is the main category / collection?